src/subcommand/wallet/transaction_builder.rs

```diff
@@ -36,6 +36,7 @@ use {
 pub(crate) enum Error {
   NotInWallet(SatPoint),
   NotEnoughCardinalUtxos,
+  UtxoContainsAdditionalInscription((SatPoint, SatPoint)),
```

Comment:
Should mention the ID of the inscription that would be sent:

```suggestion
  UtxoContainsAdditionalInscription((SatPoint, InscriptionId, SatPoint)),
```